src/i18n/locales/en/translation.json

```diff
@@ -260,6 +260,9 @@
       "label_max_cj_fee_rel": "Relative limit (per collaborator)",
       "description_max_cj_fee_rel": "The maximum fee you are willing to pay per collaborator, as a percentage of the transaction amount. Example: if you send 2 million sats and the maximum fee is set to 0.1%, you will at most pay 2,000 sats to a single collaborator.",
       "feedback_invalid_max_cj_fee_rel": "Please provide a valid maximum relative fee between {{ min }} and {{ max }}.",
+      "label_max_sweep_fee_change": "Collaborative sweep fee change",
+      "description_max_sweep_fee_change": "The maximum fee increase you are willing to pay on collaborative sweep transactions, as a percentage of the estimated transaction fee. Example: when set to 60% with 10 sats/vByte, the actual fee for collaborative sweeps can be between 4 and 16 sats/vByte. Default: {{ defaultValue }}",
```

Comment:
nit
```suggestion
      "description_max_sweep_fee_change": "The maximum fee change you are willing to pay on collaborative sweep transactions, as a percentage of the estimated transaction fee. Example: when set to 60% with 10 sats/vByte, the actual fee for collaborative sweeps can be between 4 and 16 sats/vByte. Default: {{ defaultValue }}",
```

Reply:
Deliberately changed it to "increase" instead of "change", as I thought it might be confusing to users (why should it decrease?). However, as the following example indicates the possibility of a decreasing value, I am okay either way.